UnitsNet.Tests/UnitParserTests.cs

```diff
@@ -144,6 +144,13 @@ public void Parse_MappedCustomUnit()
             Assert.Equal(HowMuchUnit.Some, parsedUnit);
         }
         
+        [Fact]
+        public void Parse_LengthUnit_MM_ThrowsExceptionDescribingTheAmbiguity()
+        {
+            var ex = Assert.Throws<AmbiguousUnitParseException>(() => UnitsNetSetup.Default.UnitParser.Parse<LengthUnit>("MM"));
+            Assert.Equal("""Cannot parse "MM" since it matches multiple units: Megameter ("Mm"), Millimeter ("mm").""", ex.Message);
+        }
```

Comment:
Notice the new message format, the other one seemed excessively long..

Reply:
I like it